src/options/vars.rs

```diff
@@ -148,6 +148,5 @@ impl MockVars {
             "NO_COLOR" => self.no_colors = value.clone(),
             _ => (),
         };
-        ()
```

Comment:
Wonder why this was even here in the first place when the function doesn't return anything :woman_shrugging: